.changeset/thirty-panthers-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-github-deployments': minor
```

Comment:
Sorry but while we're on 0.x, semver actually dictates patch level bumps for non breaking changes rather than minor :)

Reply:
Done